waku/node/v2/waku_types.nim

```diff
@@ -32,11 +36,29 @@ type
 
   QueryHandlerFunc* = proc(response: HistoryResponse) {.gcsafe, closure.}
 
+
+  Index* = object
+    ## This type contains the  description of an index used in the pagination of waku messages
+    digest*: string
```

Comment:
I would hold this value here as `MDigest[256]`, and send it over the wire as byte array.

Reply:
Sure, done